store/src/main/java/org/apache/rocketmq/store/stats/BrokerStatsManager.java

```diff
@@ -29,37 +30,37 @@
-    public static final String GROUP_GET_SIZE = "GROUP_GET_SIZE";
-    public static final String SNDBCK_PUT_NUMS = "SNDBCK_PUT_NUMS";
-    public static final String BROKER_PUT_NUMS = "BROKER_PUT_NUMS";
-    public static final String BROKER_GET_NUMS = "BROKER_GET_NUMS";
-    public static final String GROUP_GET_FROM_DISK_NUMS = "GROUP_GET_FROM_DISK_NUMS";
-    public static final String GROUP_GET_FROM_DISK_SIZE = "GROUP_GET_FROM_DISK_SIZE";
-    public static final String BROKER_GET_FROM_DISK_NUMS = "BROKER_GET_FROM_DISK_NUMS";
-    public static final String BROKER_GET_FROM_DISK_SIZE = "BROKER_GET_FROM_DISK_SIZE";
+    public static final String QUEUE_PUT_NUMS = Stats.QUEUE_PUT_NUMS;
+    public static final String QUEUE_PUT_SIZE = Stats.QUEUE_PUT_SIZE;
+    public static final String QUEUE_GET_NUMS = Stats.QUEUE_GET_NUMS;
+    public static final String QUEUE_GET_SIZE = Stats.QUEUE_GET_SIZE;
+    public static final String TOPIC_PUT_NUMS = Stats.TOPIC_PUT_NUMS;
+    public static final String TOPIC_PUT_SIZE = Stats.TOPIC_PUT_SIZE;
+    public static final String GROUP_GET_NUMS = Stats.GROUP_GET_NUMS;
+    public static final String GROUP_GET_SIZE = Stats.GROUP_GET_SIZE;
+    public static final String SNDBCK_PUT_NUMS = Stats.SNDBCK_PUT_NUMS;
+    public static final String BROKER_PUT_NUMS = Stats.BROKER_PUT_NUMS;
+    public static final String BROKER_GET_NUMS = Stats.BROKER_GET_NUMS;
+    public static final String GROUP_GET_FROM_DISK_NUMS = Stats.GROUP_GET_FROM_DISK_NUMS;
+    public static final String GROUP_GET_FROM_DISK_SIZE = Stats.GROUP_GET_FROM_DISK_SIZE;
+    public static final String BROKER_GET_FROM_DISK_NUMS = Stats.BROKER_GET_FROM_DISK_NUMS;
+    public static final String BROKER_GET_FROM_DISK_SIZE = Stats.BROKER_GET_FROM_DISK_SIZE;
     // For commercial
-    public static final String COMMERCIAL_SEND_TIMES = "COMMERCIAL_SEND_TIMES";
-    public static final String COMMERCIAL_SNDBCK_TIMES = "COMMERCIAL_SNDBCK_TIMES";
-    public static final String COMMERCIAL_RCV_TIMES = "COMMERCIAL_RCV_TIMES";
-    public static final String COMMERCIAL_RCV_EPOLLS = "COMMERCIAL_RCV_EPOLLS";
-    public static final String COMMERCIAL_SEND_SIZE = "COMMERCIAL_SEND_SIZE";
-    public static final String COMMERCIAL_RCV_SIZE = "COMMERCIAL_RCV_SIZE";
-    public static final String COMMERCIAL_PERM_FAILURES = "COMMERCIAL_PERM_FAILURES";
+    public static final String COMMERCIAL_SEND_TIMES = Stats.COMMERCIAL_SEND_TIMES;
+    public static final String COMMERCIAL_SNDBCK_TIMES = Stats.COMMERCIAL_SNDBCK_TIMES;
+    public static final String COMMERCIAL_RCV_TIMES = Stats.COMMERCIAL_RCV_TIMES;
+    public static final String COMMERCIAL_RCV_EPOLLS = Stats.COMMERCIAL_RCV_EPOLLS;
+    public static final String COMMERCIAL_SEND_SIZE = Stats.COMMERCIAL_SEND_SIZE;
+    public static final String COMMERCIAL_RCV_SIZE = Stats.COMMERCIAL_RCV_SIZE;
+    public static final String COMMERCIAL_PERM_FAILURES = Stats.COMMERCIAL_PERM_FAILURES;
```

Comment:
IMO these definition are duplicate, maybe use Stats.xxx directly is better.

Reply:
These public fields are used by external projects such as rocketmq-exporter, I will chage to deprecated first.